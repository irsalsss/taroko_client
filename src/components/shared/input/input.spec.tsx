
import { render } from '@/utils/test/test'
import Input from './input';

describe("Input", () => {
  it("should render input successfully", () => {
    const { baseElement } = render(<Input placeholder="Search ..." />)

    expect(baseElement).toMatchSnapshot();
    expect(baseElement).toBeTruthy();
  });
});
