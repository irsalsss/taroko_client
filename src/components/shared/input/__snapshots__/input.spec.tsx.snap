// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render input successfully 1`] = `
<body>
  <div>
    <div
      class="relative"
    >
      <input
        class="input"
        placeholder="Search ..."
      />
    </div>
  </div>
</body>
`;
